### PR TITLE
274 notyfikacja window sensor

### PR DESCRIPTION
**Pull request by szymoneiro**
Notyfikacje do czujników otwarcia okien

### CODE REVIEW

File: substations/04_magneticHallSensor/04_magneticHallSensor.ino [hunk 1]
```diff
@@ -1,34 +1,27 @@
...
+    network.init();
 }
 
 void loop()
 {
+    network.loop();
     mqtt.loop();
     mqtt.send(magnetic_digital(digitalPin));
-    delay(100);
+    delay(1000);
```
Comment (Siwy160) [inline substations/04_magneticHallSensor/04_magneticHallSensor.ino:26]: @kuba8686 Did it differently. See at https://github.com/intive/patronage20-embedded/pull/34/files#diff-1419b57635f37de432472145b2a3a875
Here you are blocking whole loop function for 1 second. So potential  information received from sensors could be lost.

Comment (szymoneiro) [reply substations/04_magneticHallSensor/04_magneticHallSensor.ino]: fixed

File: substations/04_magneticHallSensor/04_magneticHallSensor.ino [hunk 2]
```diff
@@ -23,5 +23,5 @@ void loop()
     network.loop();
     mqtt.loop();
     mqtt.send(magnetic_digital(digitalPin));
-    delay(1000);
+    delay(1);
```
Comment (kuba8686) [inline substations/04_magneticHallSensor/04_magneticHallSensor.ino:26]: still not fully correct. 
Right now Your subst will notify status 1000 times per second. 
Try like this:
keep delay(1), insert a global variable ex int notificaller = 0; then put at the end of a loop:

   ```
 if (++notifCaller == 1000) {
        mqtt.send(magnetic_digital(digitalPin));
        notifCaller = 0;
    }
```

Comment (szymoneiro) [reply substations/04_magneticHallSensor/04_magneticHallSensor.ino:29]: Okay, set it as you requested and after testing it is working properly now

Comment (kuba8686) [reply substations/04_magneticHallSensor/04_magneticHallSensor.ino:29]: another change, that appear after reviev from Michał to my PR. Instead ++notifCaller == 1000 do ++notifCaller >= 1000 and make notifiCaler unsigned int. Explanation why You will find in my old pull request about BlindsController (#34), at Michal comment.

Comment (kuba8686) [reply substations/04_magneticHallSensor/04_magneticHallSensor.ino:29]: another change, that appear after reviev from Michał to my PR. Instead ++notifCaller == 1000 do ++notifCaller >= 1000 and make notifiCaler unsigned int. Explanation why You will find in my old pull request about BlindsController (#34), at Michal comment.

Comment (szymoneiro) [reply substations/04_magneticHallSensor/04_magneticHallSensor.ino:29]: done

**Pull request merged**